### PR TITLE
Add pages for spring training events

### PR DESCRIPTION
**Pull request by xamberl**
Preview page (when it generates): https://chtc.github.io/web-preview/preview-spring-2025-training/events.html

### CODE REVIEW

File: _events/2025-01-14-information-session.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+{% capture main %}
+
+<p style="font-size: larger; font-weight: bold;">Learn about CHTC's computing 
+and data services for UW-Madison affiliates at this information session!</p>
+
+CHTC staff will present an overview of CHTC services and how these services can help 
+researchers accomplish their research and computational goals. CHTC staff will also 
+help attendees identify the next steps for getting started - whether that is 
+getting an account, how to log in, or how to start running work. After
```
Comment (ChristinaLK) [inline _events/2025-01-14-information-session.md:24]:
```suggestion
getting an account, how to log in, or how to start running work. If you are attending in person, after 
```

File: _events/2025-02-11-containers.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+[Register Here](https://docs.google.com/forms/d/e/1FAIpQLSdmkVtOBtocgX57RmShjhzRuhNAeuBhU8iUXOXOWLY6AQUAug/viewform)
+
+{% endcapture %}
+
+{% capture subsection %}
+{: .mt-0 }
+
+### Who
+
+* CHTC users who need to install software on our High Throughput or High Performance Computing system
```
Comment (ChristinaLK) [inline _events/2025-02-11-containers.md:32]: @aowen-uwmad are you comfortable with both HPC + HTC?

File: _events/2025-02-11-containers.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+### When
+
+Tuesday, April 8, 10:30am - 12:00pm
```
Comment (ChristinaLK) [inline _events/2025-02-11-containers.md:36]:
```suggestion
Tuesday, February 11, 10:30am - 12:00pm
```

**Review by ChristinaLK — COMMENTED** [review]
* For the workshops, do we want to add a question to the form that confirms people have an account? 
* Date of the March event is missing the leading 0 of the month in the file name. 
* It would be nice to have some picture variety between the workshops. I either search the campus photo library for generic photos or I go through my collection of nice "facilitator-y" images. Will share some links.

**Comment by xamberl** [comment]
- Updated the registration forms for the February and April workshop to ask if they have CHTC accounts.
- Updated events with new images for some variety
- Fixed the name of the March event markdown file

**Review by ChristinaLK — APPROVED** [review]
Looks great!!! 😍

**Pull request merged**